app/stores/AppStore.js

```diff
@@ -96,21 +109,23 @@ export default class AppStore extends Store {
     this.getTermsOfUseAcceptanceRequest.execute();
   };
 
+  _getSendLogsChoice = async () => {
+    const sendLogs = await this.getSendLogsChoiceRequest.execute().promise;
+    ipcRenderer.send('send-logs-choice', sendLogs);
```

Comment:
So then i would move this into its own method called `sendLogsChoiceToMainProcess` and call it separately in the `setup` method after `_getSendLogsChoice`. One very important rule for me is that getters don't have side-effects, as this leads to big confusion and bad design.